pkg/volume/volume_linux_test.go

```diff
@@ -296,14 +309,15 @@ func TestSetVolumeOwnershipMode(t *testing.T) {
 				t.Fatalf("error reading permission stats for tmpdir: %s", tmpDir)
 			}
 
-			var expectedGid int64 = int64(stat.Gid)
+			var expectedGID = int64(stat.Gid)
 			err = test.setupFunc(tmpDir)
 			if err != nil {
 				t.Errorf("for %s error running setup with: %v", test.description, err)
 			}
 
 			mounter := &localFakeMounter{path: "FAKE_DIR_DOESNT_EXIST"} // SetVolumeOwnership() must rely on tmpDir
-			err = SetVolumeOwnership(mounter, tmpDir, &expectedGid, test.fsGroupChangePolicy, nil)
+			ownershipChanger := NewVolumeOwnership(mounter, tmpDir, &expectedGID, test.fsGroupChangePolicy, nil)
+			err = ownershipChanger.ChangePermissions()
```

Comment:
This change is causing the test to fail when run as root, we need to revert it or fix it to work regardless of which user runs the test.

```
{Failed  === RUN   TestSetVolumeOwnershipOwner/*fsGroup=3000
    volume_linux_test.go:592: for *fsGroup=3000 error verifying permissions with: invalid owner on /tmp/volume_linux_ownership3055383160/file.txt
--- FAIL: TestSetVolumeOwnershipOwner/*fsGroup=3000 (0.60s)

=== RUN   TestSetVolumeOwnershipOwner/symlink
    volume_linux_test.go:592: for symlink error verifying permissions with: invalid owner on /tmp/volume_linux_ownership3642780172/file_link.txt
--- FAIL: TestSetVolumeOwnershipOwner/symlink (0.90s)

=== RUN   TestSetVolumeOwnershipOwner
--- FAIL: TestSetVolumeOwnershipOwner (1.51s)
}
```

cc: @liggitt @chrishenzie